integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -187,6 +188,13 @@ public Long load(String groupName) {
     mIsUserGroupTranslation = conf.getBoolean(PropertyKey.FUSE_USER_GROUP_TRANSLATION_ENABLED);
     mMaxUmountWaitTime = (int) conf.getMs(PropertyKey.FUSE_UMOUNT_TIMEOUT);
     mAuthPolicy = AuthPolicyFactory.create(mFileSystem, conf, this);
+    if (opts.isDebug()) {
+      try {
+        LogUtils.setLogLevel("alluxio.fuse.AlluxioJniFuseFileSystem", "DEBUG");
```

Comment:
Can we avoid hard coding this name?
I think there may be some ways to get the full name with package
Have you tried some other methods? e.g.this.getClass().getCanonicalName()
from https://stackoverflow.com/questions/9729197/how-to-get-current-class-name-including-package-name-in-java